scripts/exp_config.js

```diff
@@ -120,7 +120,7 @@ function directoryExists(path) {
 module.exports = function(context) {
   //get platform from the context supplied by cordova
  
-  console.log("Adding the ForeSee SDK config file");
+  console.log("Adding the Verint SDK config file");
```

Comment:
This might be better as 
```suggestion
  console.log("Adding the Verint XM config file");
```